src/components/Button.js

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types';
+
+function Button({ btnName }) {
+  return (
+    <button type="button">
+      { btnName }
+    </button>
+  );
+}
+
+Button.propTypes = {
+  btnName: PropTypes.string,
+};
+
+Button.defaultProps = {
+  btnName: '',
+};
+
+export default Button;
```

Comment:
## Fix `Button` component
- [ ] The incoming prop `btnName` is foreign to `React`. It doesn't know where is it coming from. And since you are exporting and importing this component function inside of the `./src/component/ButtonPanel.js` component, there is a need to change the props naming either here or in the `ButtonPanel` function component where you are calling it so as to match the prop parameters getting passed to it.

```suggestion
import PropTypes from 'prop-types';

function Button({ name }) {
  return (
    <button type="button">
      { name }
    </button>
  );
}

Button.propTypes = {
  name: PropTypes.string,
};

Button.defaultProps = {
  name: '',
};

export default Button;
```
Below is what the user currently sees on the page 👇🏾 

|Expected Page|Current Page|
|:--:|:--:|
|![expected-view](https://user-images.githubusercontent.com/22345961/115112277-778f0980-9f8d-11eb-9d26-95d10bcc53e7.png)|![current-view](https://user-images.githubusercontent.com/22345961/115112279-7b229080-9f8d-11eb-8c65-4a1ebce27c54.png)|